dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java

```diff
@@ -237,7 +246,64 @@
         typeMap.put("Geolocation", TypeUtil.createObjectType("Geolocation",
                 geoFields));
 
+        final Map<String, TypeFetcher> layoutFields = new HashMap<>();
+        layoutFields.put("width", new TypeFetcher(GraphQLLong,
```

Comment:
Codacy found an issue: [The String literal "width" appears 4 times in this file; the first occurrence is on line 250](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6084048)